packages/cli/src/services/workflow-statistics.service.ts

```diff
@@ -90,7 +90,7 @@ export class WorkflowStatisticsService extends TypedEmitter<WorkflowStatisticsEv
 				}
 			}
 		} catch (error) {
-			this.logger.verbose('Unable to fire first workflow success telemetry event');
+			this.logger.debug('Unable to fire first workflow success telemetry event');
```

Comment:
Shouldn't we fire a sentry alert in cases like these? Nobody probably ever notices if the code ends up here and this might be hiding a bug. Not saying it should be done in this PR, but not delighted by this type of pattern

Reply:
Agree, the workflow statistics service needs a lot of work. Let's do a follow-up.